front/app/containers/Admin/tools/routes.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { lazy } from 'react';
+import moduleConfiguration from 'modules';
+import PageLoading from 'components/UI/PageLoading';
+import messages from './messages';
+import Outlet from 'components/Outlet';
+import { Outlet as RouterOutlet } from 'react-router-dom';
+import HelmetIntl from 'components/HelmetIntl';
+
+const Tools = lazy(() => import('.'));
```

Comment:
I don't think lazy loading will have an effect here, since this component is visible on page load. No?

Reply:
It can still be useful even for components that are visible but we can ignore in this case, since this is not resource intensive